full/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Read the Docs - Environment base
+FROM readthedocs/base:latest
+MAINTAINER Read the Docs <support@readthedocs.com>
+LABEL version="2.0"
+
+# from readthedocs.common
+RUN apt-get -y install libpq-dev libxml2-dev libxslt-dev libxslt1-dev postgresql-client libmysqlclient-dev
+
+# from readthedocs.build
+RUN apt-get -y install libfreetype6 g++ sqlite libevent-dev libffi-dev \
```

Comment:
Lumping all dependencies into one image is a bad idea. If there is a problem installing any one of these, the entire image is hosed and needs to be restarted. It's not common, but in development I hit this a number of times.